src/ai_hawk/linkedIn_easy_applier.py

```diff
@@ -692,7 +694,13 @@ def _find_and_handle_radio_question(self, section: WebElement) -> bool:
             question_text = section.text.lower()
             options = [radio.text.lower() for radio in radios]
 
-            existing_answer = self._find_existing_answer(question_text)
+            existing_answer = None
```

Comment:
why is this going reverse of clean code?

Reply:
_find_existing_answer(question_text) - is printing unnecessary log - for every iteration( question ) that is present in answers.json